### PR TITLE
Test for #20657 SIMD byref bug

### PR DESCRIPTION
**Pull request by CarolEidt**

### CODE REVIEW

**Comment by CarolEidt** [comment]
@dotnet/jit-contrib PTAL

**Comment by CarolEidt** [comment]
I verified that this fails if I revert the fix in #19857

File: tests/src/JIT/Regression/JitBlue/GitHub_20657/GitHub_20657.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Numerics;
+
+// GitHub 20657:
```
Comment (BruceForstall) [inline tests/src/JIT/Regression/JitBlue/GitHub_20657/GitHub_20657.cs]: Did you mean to include a description of the issue here? That was my assumption from the trailing colon.

fwiw, though I generally like issue descriptions in the tests, it not always clear how useful they are.

**Comment by CarolEidt** [comment]
test Windows_NT arm64 Cross Checked Innerloop Build and Test

**Pull request merged**